explainability-service/src/main/java/org/kie/trustyai/service/endpoints/consumer/CloudEventConsumer.java

```diff
@@ -36,12 +40,19 @@ public void consumeKubeflowRequest(CloudEvent<byte[]> cloudEvent) {
 
     @Funq
     @CloudEventMapping(trigger = "org.kubeflow.serving.inference.response")
-    public void consumeKubeflowResponse(CloudEvent<InferenceLoggerOutput> cloudEvent) {
+    public void consumeKubeflowResponse(CloudEvent<byte[]> cloudEvent) throws JsonProcessingException {
         LOG.debug("Received Kubeflow response with id = " + cloudEvent.id());
+
         final KServeOutputPayload output = new KServeOutputPayload();
         output.setId(cloudEvent.id());
         output.setModelId(cloudEvent.extensions().get("Inferenceservicename"));
-        output.setData(cloudEvent.data());
+        byte[] original = cloudEvent.data();
+        String decoded = new String(original, StandardCharsets.UTF_8);
```

Comment:
If the assumption is that `byte[]` in reality is a `String`, what about using `CloudEvent<String>` directly? 
I guess the default deserialization as `String` in Jackson is UTF_8 already

Reply:
@danielezonca I've moved this to a [separate issue](https://github.com/trustyai-explainability/trustyai-explainability/issues/658) if that's ok